.github/workflows/ci.yaml

```diff
@@ -52,38 +52,38 @@ jobs:
           composer remove --no-update --dev --no-interaction --no-progress \
             phan/phan phpstan/phpstan vimeo/psalm \
             infection/infection friendsofphp/php-cs-fixer
-          composer update --prefer-dist --no-interaction --no-progress ${{ matrix.dependencies }}
+          composer update --prefer-dist --no-interaction --no-progress
 
       - name: Execute tests
         run: |
-          php vendor/bin/phpunit
+          ./vendor/bin/phpunit
 
   validate:
     name: Static Analysis and Validation
     runs-on: ubuntu-latest
-
-    env:
-      PHP_VERSION: '8.2'
+    strategy:
+      matrix:
+        php-version: ['8.2']
```

Comment:
Always use a matrix.

Reply:
I'm confused: in one case you remove the matrix because "a single value does not make a matrix" and in another place you introduce the matrix with a single value :thinking:

Reply:
Yeah, I think it actually runs PHP CS Fixer another time before the static analysis. The reason there's another stage for PHP CS Fixer because I want to be able to see that there are coding style issues right away, and last time I checked this was the fastest way to do just the code style check.